Changelog.md

```diff
@@ -7,6 +7,7 @@ __New Features__
 - LightingGroups can now be specified using kWh/year annual consumption values as an alternative to fractions of different lighting types.
 - Allows modeling one or more occupant vacancy periods (`VacancyPeriods` in the HPXML file).
 - ReportSimulationOutput measure: Allows specifying the number of decimal places for timeseries output.
+- Adds new component loads outputs `Windows Conduction`, `Windows Solar`, `Skylights Conduction`, `Skylights Solar`, `Lighting`, and `Residual`, removes `Windows`, `Skylights`, and `Internal Mass` components.
```

Comment:
Software applications may be using the current component load types, so we need to mark this as a breaking change. Let's also try to make it a bit clearer what the changes are.

```suggestion
- **Breaking change**: Updates component loads outputs:
  - Replaces `Windows` and `Skylights` with `Windows Conduction`, `Windows Solar`, `Skylights Conduction`, and `Skylights Solar`.
  - Disaggregates `Lighting` from `Internal Gains`.
  - Adds `Residual` (calculated as difference between energy delivered and sum of component loads).
  - Removes `Internal Mass`.
```

Reply:
Also, the documentation needs to be updated. See https://openstudio-hpxml.readthedocs.io/en/latest/workflow_outputs.html#annual-component-building-loads